actions/access_keys/routes.go

```diff
@@ -22,8 +22,8 @@ func OldAccessKeysHandler(appConfig *config.AppConfig, services *config.AppServi
 		accessKeyGroup.GET("", action.oldGet)
 		accessKeyGroup.DELETE("", action.oldRevoke)
 		accessKeyGroup.POST("/count", action.count)
-		accessKeyGroup.GET("/search", action.search)
-		accessKeyGroup.POST("/search", action.search)
+		accessKeyGroup.GET("/search", action.oldSearch)
```

Comment:
I don't know why it's there but it should be probably removed, because we didn't handle GET in old search endpoint

Reply:
removed